setup.py

```diff
@@ -16,7 +16,7 @@
     "urllib3>=1.26,<3",
     "eve>=1.1.2,<=2.1.0",
     "eve-elastic>=7.4.0,<7.5.0",
-    "elasticsearch<7.14",  # we are using oss version on test server
+    "elasticsearch[async]<7.14",  # we are using oss version on test server
```

Comment:
we're using 7.17 now on test server so if needed you can use more recent version